src/sentry/integrations/slack/endpoints/command.py

```diff
@@ -72,8 +80,19 @@ def reply(self, slack_request: SlackRequest, message: str) -> Response:
             }
         )
 
-    def link_team(self, slack_request: SlackCommandRequest) -> Response:
+    def get_team_linked_to_channel(
```

Comment:
When naming a function like this I like to use the `get_` prefix when I'll be returning the object or value. We can do that here and infer existence by it not being `None`. If you want to return a boolean I'd name this with the `is_` prefix. Something like: `is_team_linked_to_channel()`.